website/src/components/Chat/ChatMessageEntry.tsx

```diff
@@ -92,7 +92,9 @@ export const ChatMessageEntry = memo(function ChatMessageEntry({
 
   const isAssistant = message.role === "assistant";
   const [isEditing, setIsEditing] = useBoolean(false);
+  const [isPlainText, setIsPlainText] = useState<boolean>(false);
   const inputRef = useRef<HTMLTextAreaElement>(null);
+  const memoIsPlainText = useMemo(() => isPlainText, [isPlainText]);
```

Comment:
This probably has no effect, you can remove this.

Reply:
noted. i have made the change.